src/pages/home/Home.js

```diff
@@ -41,59 +43,57 @@ class Home extends Component {
   };
 
   render() {
-    const { t } = this.props;
     const { legal } = this.state;
 
     return (
       <Fragment>
         <Navbar />
         <Hero onClick={this.handleOnClickCTA} />
-        <main className={styles.content}>
-          <HomeSection
-            id="about"
-            title={t('home-about-title')}
-            text={t('home-about-text')}
-            media={<Image src={ImageAbout} alt={t('home-about-media-alt')} />}
-          />
-          <HomeSection
-            id="mission"
-            title={t('home-mission-title')}
-            text={t('home-mission-text')}
-            media={<Image src={ImageMission} alt={t('home-mission-media-alt')} />}
-          />
-          <HomeSection
-            id="contact"
-            title={t('home-contact-title')}
-            media={<img src={ImageContact} alt={t('home-contact-media-alt')} />}
-          >
-            <p>
-              <Interpolate
-                i18nKey="home-contact-text"
-                slackOrg={
-                  <a
-                    href={config.contact.SLACK_URL}
-                    className={styles.slackOrg}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {t('slack-org')}
-                  </a>
-                }
-                email={
-                  <a
-                    target="_blank"
-                    href={`mailto:${config.contact.EMAIL}`}
-                    className={styles.email}
-                    rel="noopener noreferrer"
-                  >
-                    {config.contact.EMAIL}
-                  </a>
-                }
+        <I18n>
+          {({ i18n }) => (
+            <main className={styles.content}>
+              <HomeSection
+                id="about"
+                title={i18n._(t`home.about.title`)}
+                text={i18n._(t`home.about.description`)}
+                media={<Image src={ImageAbout} alt={i18n._(t`home.about.img`)} />}
+              />
+              <HomeSection
+                id="mission"
+                title={i18n._(t`home.mission.title`)}
+                text={i18n._(t`home.mission.description`)}
+                media={<Image src={ImageMission} alt={i18n._(t`home.mission.img`)} />}
               />
-            </p>
-            <SocialMedia />
-          </HomeSection>
-        </main>
+              <HomeSection
+                id="contact"
+                title={i18n._(t`home.contact.title`)}
+                media={<img src={ImageContact} alt={i18n._(t`home.contact.img`)} />}
+              >
+                <p>
+                  <Trans id="home.contact.description">
+                    <a
+                      href={config.contact.SLACK_URL}
+                      className={styles.slackOrg}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      Slack Organization
```

Comment:
do we want a variable fort this too?

Reply:
Yeah, it gets replaced by the macro at build time (Line 73)